### PR TITLE
Testing To Do list: part 1

### PR DESCRIPTION
**Pull request by lu-jim**
The First commit of the week. The purpose of this section is to test the add and remove item functionalities.
# Changes
- Created test file and installed Jest
- Installed Babel to convert ES6 to CommonJS
- Tested the adding feature
- Tested the delete feature

### CODE REVIEW

File: todoList.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/**
+ * @jest-environment jsdom
```
Comment (SajjadAhmad14) [inline todoList.test.js:2]: Please add the `npm run test` command in the README file.

File: todoList.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+describe('Add Item', () => {
+  it('Add item to the DOM', () => {
+    document.body.innerHTML = '<div> <ul id="to-do"> </div>';
+    todoList.addTask(new ToDo('Finish the project', 0));
+    const list = document.querySelectorAll('#to-do li');
+    expect(list).toHaveLength(1);
+  });
+
+  it('Add item to the local Storage', () => {
```
Comment (SajjadAhmad14) [inline todoList.test.js:21]: Please use `mock` for this function which is used to add items to localStorage.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Luis and Rajendra, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: todoList.test.js (hunk 2 above)
Comment (Medini-Rajendra) [reply todoList.test.js:21]: Fixed this problem.

File: todoList.test.js (hunk 1 above)
Comment (Medini-Rajendra) [reply todoList.test.js:2]: added npm run test command to Readme File

**Review by D4nielJ — APPROVED** [review]
## Hello @lu-jim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**